examples/example-gateway/build/clients/baz/baz.go

```diff
@@ -0,0 +1,166 @@
+// Code generated by zanzibar
+// @generated
+
+// Package bazClient is generated code used to make or handle TChannel calls using Thrift.
+package bazClient
+
+import (
+	"context"
+	"errors"
+	"strconv"
+	"time"
+
+	"github.com/uber/zanzibar/runtime"
+
+	clientsBazBaz "github.com/uber/zanzibar/examples/example-gateway/build/gen-code/clients/baz/baz"
+)
+
+// Interfaces for the service and client for the services defined in the IDL.
+
+// TChanBaz is the interface that defines the server handler and client interface.
+type TChanBaz interface {
+	Call(
+		ctx context.Context,
+		reqHeaders map[string]string,
+		args *clientsBazBaz.SimpleService_Call_Args,
+	) (
+		*clientsBazBaz.BazResponse,
+		map[string]string,
+		error,
+	)
+
+	Simple(
+		ctx context.Context,
+		reqHeaders map[string]string,
+	) (
+		map[string]string,
+		error,
+	)
+
+	SimpleFuture(
+		ctx context.Context,
+		reqHeaders map[string]string,
+	) (
+		map[string]string,
+		error,
+	)
+}
+
+// NewClient returns a new TChannel client for service baz.
+func NewClient(gateway *zanzibar.Gateway) *BazClient {
+	serviceName := gateway.Config.MustGetString("clients.baz.serviceName")
+	sc := gateway.Channel.GetSubChannel(serviceName)
+
+	ip := gateway.Config.MustGetString("clients.baz.ip")
```

Comment:
added a TODO in https://github.com/uber/zanzibar/pull/57/files#diff-b99ff0337e82249d23a0d1420b13207fR60